setup.py

```diff
@@ -50,10 +50,12 @@
                 'dali-dataset',
             ],
             'docs': [
+                'DALI-dataset>=1.0.0',
```

Comment:
does dali-dataset need to also be here? it's already in docs/requirements.txt

Reply:
good question. i'm not sure. i tried the following:

`pip install .[docs]`
`cd docs`
`make html`

and it passed. it didn't pass without `DALI-dataset`. But perhaps i'm making docs the wrong way? How is `docs/requirements.txt` linked exactly?

Reply:
I'm not sure either - it gets picked up by readthedocs. Let's merge as-is, it doesn't really hurt to have an extra requirement in the docs build.